pytorch_blade/scripts/pip/requirements-dev-pre+cu117.txt

```diff
@@ -6,6 +6,6 @@ hypothesis
 expecttest
 aliyun-log-python-sdk==0.6.48.6
 cryptography
-torch==1.14.0.dev20221028+cu117
-torchvision==0.15.0.dev20221028+cu117
+torch
```

Comment:
a fixed version can avoid some unexpected compile or running errors.

Reply:
but fixing to a version is not a test for `pre` ?

Reply:
I think @Yancey1989 's opinion is that we should create a PR to update the nightly torch version daily.

Reply:
done